### PR TITLE
Add Python and Go write instructions to Cloud Dedicated get started

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 1]
```diff
@@ -135,101 +135,24 @@ home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
 
 ## Write line protocol to InfluxDB
 
-Use the **InfluxDB UI**, **`influx` CLI**, or **InfluxDB API** to write the
+Use the **InfluxDB v2 HTTP API** or **InfluxDB client libraries** to write the
 line protocol above to InfluxDB.
+Examples are provided for the **Python** and **Go** client libraries.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/get-started/write.md:140]:
```suggestion
The following examples show how to use the **Python** and **Go** client libraries to write line protocol.
```

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 2]
```diff
@@ -289,6 +212,254 @@ home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
...
+	"log"
+	"os"
+	"time"
+
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+)
+
+func dbWrite(ctx context.Context) error {
+	// Create write client
+	url := "https://us-east-1-1.aws.cloud2.influxdata.com"
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/get-started/write.md:395]: Should this be `https://cloud2.influxdata.com`?

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 3]
```diff
@@ -289,6 +212,254 @@ home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
...
+	token := os.Getenv("INFLUX_TOKEN")
+	writeClient := influxdb2.NewClientWithOptions(url, token, influxdb2.DefaultOptions().SetPrecision(time.Second))
+
+	// Define write API
+	org := ""
+	bucket := "<YOUR_INFLUX_DATABASE>"
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/get-started/write.md:401]:
```suggestion
	bucket := "DATABASE_NAME"
```

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 4]
```diff
@@ -289,6 +212,254 @@ home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
...
+)
+
+func dbWrite(ctx context.Context) error {
+	// Create write client
+	url := "https://us-east-1-1.aws.cloud2.influxdata.com"
+	token := os.Getenv("INFLUX_TOKEN")
+	writeClient := influxdb2.NewClientWithOptions(url, token, influxdb2.DefaultOptions().SetPrecision(time.Second))
+
+	// Define write API
+	org := ""
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/get-started/write.md:400]: In my experience, `org` can't be empty.

**Review by jstirnaman — APPROVED** [review]

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 5]
```diff
@@ -289,6 +212,250 @@ home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
...
+	"log"
+	"os"
+	"time"
+
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+)
+
+func dbWrite(ctx context.Context) error {
+	// Create write client
+	url := "https://us-east-1-1.aws.cloud2.influxdata.com"
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/get-started/write.md:389]:
```suggestion
	url := "https://cloud2.influxdata.com"
```

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 6]
```diff
@@ -289,6 +212,250 @@ home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
...
+          _For security reasons, we recommend setting this as an environment
+          variable rather than including the raw token string._
+
+        Because the timestamps in the line protocol are in second
+        precision, **use `SetPrecision(time.Second)` to define the write precision option**.
+    
+    4.  Use `writeClient.WriteAPIBlocking` to define a `writeAPI`.
+        The write API requires the following credentials:
+
+        - **org**: _Leave as an empty string_
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/get-started/write.md:361]:
```suggestion
        - **org**: _Provide an arbitrary string. This credential is ignored._
```

File: content/influxdb/cloud-dedicated/get-started/write.md (hunk 4 above)
Comment (sanderson) [reply content/influxdb/cloud-dedicated/get-started/write.md:394]: K, I'll note that this should be an arbitrary string.

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 7]
```diff
@@ -289,6 +212,250 @@ home,room=Kitchen temp=22.7,hum=36.5,co=26i 1641067200
...
+)
+
+func dbWrite(ctx context.Context) error {
+	// Create write client
+	url := "https://cloud2.influxdata.com"
+	token := os.Getenv("INFLUX_TOKEN")
+	writeClient := influxdb2.NewClientWithOptions(url, token, influxdb2.DefaultOptions().SetPrecision(time.Second))
+
+	// Define write API
+	org := ""
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/get-started/write.md:394]:
```suggestion
	org := "ignored"
```

**Pull request merged**